packages/wasm/src/modules/network-manager.ts

```diff
@@ -14,11 +15,13 @@ export const SUBSTRATE_NETWORKS : Record<NetworkId, NetworkInfo> = {
     name: 'Dock PoS Mainnet',
     substrateUrl: ['wss://mainnet-node.dock.io', 'wss://mainnet-node-2.dock.io'],
     addressPrefix: 22,
+    cheqdApiUrl: 'https://cheqd-mainnet-rpc.autostake.com/',
```

Comment:
By default the wallet can't connect to multiple networks at the same time, the network resolver and switch logic would not work properly. To avoid a major refactoring I added substrate and cheqd into the same entry. And the wallet keep with the `mainnet`, and `testnet` networks as we had before.